### ISSUE TITLE
Sidebar stop working with Fuse 1.0.1

### ISSUE THREAD

**Issue by jesusmartinoza** [Duckers/Fuse.MaterialDesign#11]
Hi guys, I've been using this package for a while and now doesn't work anymore with last Fuse version 1.0.1 Doesn't open the drawer or the dots menu. Here is a video:

[![Fuse.MaterialDesign issues](http://img.youtube.com/vi/yGK-9cfrC6A/0.jpg)](https://youtu.be/yGK-9cfrC6A)

Fuse Studio Problems tab outputs this:
```
{onPressed} not found in data context
{onReleased} not found in data context
{onPlaced} not found in data context
{onPressed} not found in data context
{onReleased} not found in data context
{onPlaced} not found in data context
{onPressed} not found in data context
{onReleased} not found in data context
{onPlaced} not found in data context
```

 Fuse version 1.0.1 (build 13566)

**Comment by eksperts** [comment]
I checked out the `release-1.0` branch on [fuselibs-public](https://github.com/fusetools/fuselibs-public), built it and ran this package. While the [PR in question](https://github.com/fusetools/fuselibs-public/pull/54) has supposedly been merged to the branch and even shipped with a production Fuse build, the problem is still there - the edge panel would not open.

The other "Problems" are just notices really, but the edge panel is something worse, it just doesn't open and fails silently.

**Comment by eksperts** [comment]
If we change [this line](https://github.com/Duckers/Fuse.MaterialDesign/blob/master/Components/DrawerPanel.ux#L18) to say
```
<Material.Drawer ux:Binding="Children" Edge="Left" />
```
(remove the `ux:Ref="Drawer"` bit),

the left edge drawer starts working again. What is it about `ux:Ref` that causes this behaviour?

**Comment by Duckers** [comment]
Investigating now

**Comment by Duckers** [comment]
Thanks for reporting! Turned out to be a regression in the UX compiler when the `ux:Test` feature was introduced. 

The fix will roll out in the next version of Fuse (1.0.3 or 1.1)

**Comment by anettegj** [comment]
A fix for this issue is now available in Fuse 1.0.3 which can be found on [here](https://www.fusetools.com/downloads/).
